packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -877,6 +890,28 @@ export class SnapController extends BaseController<
 
   readonly #preinstalledSnaps: PreinstalledSnap[] | null;
 
+  readonly #metaMetricsController: MetaMetricsController;
+
+  readonly #verifiedSnaps?: { [snapId: string]: VerifiedSnap };
+
+  readonly #trackSnapExport = throttleTracking(
+    (snapId: SnapId, handler: string, success: boolean, origin: string) => {
+      this.#metaMetricsController.trackEvent({
```

Comment:
Is it possible to use the MetaMetricsController via the messenger? Does it work across mobile and extension?

Reply:
Maybe we can just pass in the hook then? That way it could work on mobile too

Reply:
We should validate that this will work on mobile before we merge